test/runtests.jl

```diff
@@ -659,4 +659,33 @@ end
     @test RGB(Gray(0.2), Gray(0.3), Gray(0.4)) == RGB(0.2, 0.3, 0.4)
 end
 
+@testset "RGB565" begin
+    c = RGB565(31, 63, 31)  # white
+    @test blue(c) == 31
+    @test green(c) == 63
+    @test red(c) == 31
+    @test UInt16(c) == 65535
```

Comment:
```suggestion
    @test reinterpret(UInt16, c) == 65535
```
If `UInt16(c)` does the same thing, that is a serious bug and needs to be fixed.